### ISSUE TITLE
Dependency issues

### ISSUE THREAD

**Issue by rmrfslashbin** [skycocker/chromebrew#1617]
Support libs are upgraded but applications require previous versions. Both libx265 and libruby need to be version locked until ffmpeg and vim can support newer versions.

```
chronos@localhost ~ $ ffmpeg 
ffmpeg: error while loading shared libraries: libx265.so.116: cannot open shared object file: No such file or directory
chronos@localhost ~ $ vim
vim: error while loading shared libraries: libruby.so.2.4: cannot open shared object file: No such file or directory
chronos@localhost ~ $ 
```

## Versions and system information
chromebrew: `0.4.3`

ChromeOS: `chronos@localhost ~ $ cat /etc/lsb-release 
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4510202
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=22
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R64-10176.22.0
CHROMEOS_RELEASE_BUILD_NUMBER=10176
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=64
CHROMEOS_RELEASE_DESCRIPTION=10176.22.0 (Official Build) beta-channel eve 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=beta-channel
CHROMEOS_RELEASE_VERSION=10176.22.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10176.22.0`

Model: Pixelbook

Architecture: `x86_64`

## Additional information
Mention things we might need to know. Like:

What I think needs to be done:
- Lock support lib versions to requirements for applications --or--
- Explicitly require lib versions for app at build/install time

**Comment by uberhacker** [comment]
@rmrfslashbin:  Thank you for reporting these dependency issues.  The good news is there is an update to ffmpeg, so I'll try and get that processed to solve the problem with libx265 along with the vim issue.

**Comment by finwo** [comment]
So I actually wrote a whole issue for it, even though it's simply a missing dependency.
While the `gnupg` package does work without it, the `pinentry` is needed to do anything useful with password protected keys (which are pretty much all of them).

If one still bothers to read how I filled in all the details for an issue: #1663

**Comment by uberhacker** [comment]
Fixed in #2858.
